examples/feature_demo/morph_facecap.py

```diff
@@ -0,0 +1,136 @@
+"""
+Facecap animation with morph targets
+====================================
+
+This example demonstrates how to animate a model with morph targets.
+The model is a facecap model with morph targets for facial expressions (52 blend shapes).
+"""
+
+################################################################################
+# .. note::
+#
+#   To run this example, you need a model from the source repo's example
+#   folder. If you are running this example from a local copy of the code (dev
+#   install) no further actions are needed. Otherwise, you may have to replace
+#   the path below to point to the location of the model.
+
+import os
+from pathlib import Path
+
+try:
+    # modify this line if your model is located elsewhere
+    model_dir = Path(__file__).parents[1] / "data"
+except NameError:
+    # compatibility with sphinx-gallery
+    model_dir = Path(os.getcwd()).parent / "data"
+
+
+################################################################################
+# Once the path is set correctly, you can use the model as follows:
+
+# sphinx_gallery_pygfx_docs = 'animate 4s'
+# sphinx_gallery_pygfx_test = 'run'
+
+import time
+import numpy as np
+import pygfx as gfx
+from scipy import interpolate
+from wgpu.gui.auto import WgpuCanvas, run
+
+gltf_path = model_dir / "facecap.glb"
```

Comment:
Do we need a license / attribution of the source?

Reply:
The model is from [Face Cap](https://www.bannaflak.com/face-cap/documentation.html#1.5), and I've made some conversions and modifications. I'll add the attribution later.

Reply:
Updated the document and added model attribution.